src/test/java/org/openrewrite/kotlin/tree/ExtensionFunctionTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.kotlin.tree;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.kotlin.Assertions.kotlin;
+
+class ExtensionFunctionTest implements RewriteTest {
+
+    @Disabled
+    @Test
+    void extensionFunction() {
+        rewriteRun(
+          kotlin(
+            """
+              fun String.escape() = this.replace("a", "b")"
```

Comment:
```suggestion
              fun String . escape ( ) = this . replace ( "a" , "b" )"
```

Reply:
We typically add spaces where possible in the `tree` tests to assert the whitespace is preserved

Reply:
I see.